persistence/context.go

```diff
@@ -14,62 +18,100 @@ func (p PostgresContext) NewSavePoint(bytes []byte) error {
 	return nil
 }
 
+// TECHDEBT(#327): Guarantee atomicity betweens `prepareBlock`, `insertBlock` and `storeBlock` for save points & rollbacks.
 func (p PostgresContext) RollbackToSavePoint(bytes []byte) error {
 	log.Println("TODO: RollbackToSavePoint not fully implemented")
 	return p.GetTx().Rollback(context.TODO())
 }
 
-func (p PostgresContext) AppHash() ([]byte, error) {
-	log.Println("TODO: AppHash not implemented")
-	return []byte("A real app hash, I am not"), nil
-}
-
-func (p *PostgresContext) Reset() error {
-	p.txResults = nil
-	p.blockHash = ""
-	p.proposerAddr = nil
-	p.blockProtoBytes = nil
-	p.blockTxs = nil
-	return nil
+func (p *PostgresContext) ComputeAppHash() ([]byte, error) {
+	// IMPROVE(#361): Guarantee the integrity of the state
+	//                Full details in the thread from the PR review: https://github.com/pokt-network/pocket/pull/285/files?show-viewed-files=true&file-filters%5B%5D=#r1033002640
```

Comment:
This is not a permalink, it leads me to an empty files list since it has filters.

![image](https://user-images.githubusercontent.com/29378614/204395762-f6eb1fb3-7a3a-41a8-815f-be52f1e41f45.png)

Reply:
Thank you. Updated here: https://github.com/pokt-network/pocket/pull/285#discussion_r1018471719